docs/modules/ROOT/pages/tutorials/getting-started-2.adoc

```diff
@@ -0,0 +1,92 @@
+= Getting Started with Project Syn: Working with Commodore
+
+This document is the second part of the xref:tutorials/getting-started.adoc["Getting Started with Project Syn"] tutorial.
+
+== Requirements
+
+If you haven't yet, please execute the steps in the xref:tutorials/getting-started.adoc["Getting Started with Project Syn"] tutorial before attempting the instructions in this document.
+
+This tutorial assumes that you have the "k3d-lieutenant" and "k3d-steward" clusters up and running and that you have installed all required tools.
+
+In addition, create the following environment variable with the path of your private key:
+
+[source,bash]
+--
+export COMMODORE_SSH_PRIVATE_KEY=~/.ssh/id_ed25519
+--
+
+Commodore requires this variable to be able to push changes to the GitLab repositories on your behalf.
+
+== Commodore
+
+https://syn.tools/commodore/[Commodore] is the third component of the Project Syn suite of tools. It's a command-line tool that compiles the state of your clusters and pushes the changes to the GitLab projects. Steward watches those projects continuously and immediately applies the changes to your clusters, following the GitOps philosophy.
```

Comment:
The changes are actually picked up by ArgoCD. Is this an intentional fuzziness?

Reply:
Again, it's a conceptual thing. What's the purpose of Steward if Argo CD does all the job? There must be a reason why Steward in there. Yes, in the background it is Argo doing the heavy work, but the crown of flowers goes to Steward. It's about giving conceptual responsibilities to these components at the level of a Getting Started guide.

Reply:
Fair, I thought such thinking might be the case here.